Apps/Common/CompatibleOperators.h

```diff
@@ -72,6 +72,13 @@ class CompatibleOperators
     //! \param[in] fe The finite element to evaluate for
     //! \param[in] f Vector with contributions
```

Comment:
Remove this line. Also, could consider slamming the two functions into one by giving f a default value.